app/components/attribute_help_texts/show_dialog_component.html.erb

```diff
@@ -0,0 +1,33 @@
+<%=
+  render(Primer::Alpha::Dialog.new(id: dialog_id, title:, size: :large)) do |dialog|
+    dialog.with_header(variant: :large)
+
+    dialog.with_body do
+      concat(
+        render(Primer::Box.new(mt: 1, classes: "op-uc-container op-uc-container__no-permalinks")) do
+          helpers.format_text(@attribute_help_text, :help_text)
+        end
+      )
+
+      if has_attachments?
+        concat render(Primer::Beta::Heading.new(tag: :h4)) { I18n.t(:label_attachment_plural) }
+        concat helpers.list_attachments(resource_representer, inputs: { allowUploading: false })
+      end
+    end
+
+    dialog.with_footer(show_divider: true) do
+      concat(
+        render(Primer::Beta::Button.new(data: { close_dialog_id: dialog_id })) { I18n.t(:button_close) }
+      )
+
+      if allowed_to_edit?
```

Comment:
Can we please change the order of the buttons? I know, that this was like that in the Angular component as well, but it makes absolutely no sense. Also from an accessibility point of view it is confusing.